### PR TITLE
[11.0][ADD] account_invoice_ubl_email_attachment

### LINKED ISSUE #77: account_invoice_ubl: Include XML file in mail
Can the XML file included when sending mail?

### PR DESCRIPTION
**Pull request by astirpe**
With module `Account Invoice UBL`, when sending an invoice by mail, only the PDF file is attached to the email and the UBL (XML) file is embedded inside the PDF.

Some accounting software cannot read the embedded XML from a PDF, so the import is only possible by manually generating the XML file.

With this module, when sending an invoice by mail, the UBL file is also automatically included as attachment, so that 2 attachments (PDF and XML) are present in the invoice mail.

Implements the solution proposed in https://github.com/OCA/edi/issues/77#issuecomment-452628346.
Fixes https://github.com/OCA/edi/issues/77.

cc @CasVissers

### CODE REVIEW

**Comment by CasVissers-360ERP** [comment]
Translations NL:
[nl.zip](https://github.com/OCA/edi/files/3134876/nl.zip)

File: account_invoice_ubl_email_attachment/models/account_invoice.py
Comment (CasVissers-360ERP) [inline account_invoice_ubl_email_attachment/models/account_invoice.py:26]: @astirpe Can we make the file name similar to the name of the invoice (report_name)?

**Comment by mgielissen** [comment]
The module works also in v12.

File: account_invoice_ubl_email_attachment/models/account_invoice.py
Comment (astirpe) [reply account_invoice_ubl_email_attachment/models/account_invoice.py:26]: @CasVissers could you check now?

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM. Functional test and a (junior) technical review.

**Comment by astirpe** [comment]
@mgielissen would you approve this module? Once it's merged I will create a PR for V12.

**Comment by mgielissen** [comment]
> 
> 
> @mgielissen would you approve this module? Once it's merged I will create a PR for V12.

Approved

**Comment by astirpe** [comment]
@mgielissen thank you!
Could you please go to https://github.com/OCA/edi/pull/106/files, button Review Changes, then Approve?
(This is actually the way to push this module forward).

**Review by mgielissen — APPROVED** [review]

**Comment by astirpe** [comment]
@alexis-via @pedrobaeza is it ok for you to merge this module?

**Comment by astirpe** [comment]
Thanks!

**Pull request merged**
